src/styles/index.css

```diff
@@ -0,0 +1,238 @@
+.cards-container {
+  box-sizing: border-box;
+  --column-count: 1;
```

Comment:
Consider using CSS custom properties (variables) for colors and sizes throughout your styles to maintain consistency and make future changes easier.